src/main/java/usecases/suggest_title/StGatewayInputData.java

```diff
@@ -0,0 +1,4 @@
+package usecases.suggest_title;
+
+public class StGatewayInputData {
+}
```

Comment:
why does this exist

Reply:
this class was deleted because it was unnecessary